### PR TITLE
AUTH-200 : Provide a means to deploy the Identity Service without Docker or Kubernetes

### PR DESCRIPTION
**Pull request by svidrascu**

### CODE REVIEW

File: distribution/tests/endpoints.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+EXISTENT_KEYCLOAK_INSTANCES=$(ps aux | grep "standalone" | awk '{print $2}' | head -n 2 )
+echo $EXISTENT_KEYCLOAK_INSTANCES
+kill $EXISTENT_KEYCLOAK_INSTANCES
+
+cd alfresco-identity-service-$IDENTITY_VERSION/bin
+
+echo "Starting the identity service"
+/bin/bash -c './standalone.sh &'
+
+sleep 15
```
Comment (kconkas) [inline distribution/tests/endpoints.sh]: Hardcoded sleep is just asking for trouble. If this is to allow Identity Service to start up, should we not rather periodically poll `curl -v http://localhost:8080/` until it returns (or reaches a timeout)?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,74 @@ Check [prerequisites section](https://github.com/Alfresco/alfresco-dbp-deploymen
...
+  $ ./standalone.sh
+  ```
+
+  Windows bat
+  ```bash
+  > ...\alfresco-identity-service-1.1.0\bin\standalone.bat
+  ```
+  Windows powershell
+  ```bash
+  > ...\alfresco-identity-service-1.1.0\bin\standalone.ps1
```
Comment (kconkas) [inline README.md]: 👍 kudos for covering both the legacy `.bat` and more modern PowerShell

File: distribution/tests/endpoints_ps.ps1 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+function checkStatus {
+    if ($args[0] -notmatch $args[1]) {
+        Get-WmiObject Win32_Process -filter "CommandLine LIKE '%alfresco-identity-service-$IDENTITY_VERSION%'" | foreach { kill $_.ProcessId }
+        $test=$args[0]
+        $check=$args[1]
+        throw "Accessing $test does not output $check "
+    }
+}
+
+checkStatus (curl -v http://localhost:8080/auth/).StatusCode "200"
```
Comment (kconkas) [inline distribution/tests/endpoints_ps.ps1]: Does curl need to be pre-installed on a Windows box in order for this to work?

Comment (svidrascu) [reply distribution/tests/endpoints_ps.ps1]: Nope, the curl that you actually see here is an alias for Invoke-WebRequest, a command that comes with powershell. https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Utility/Invoke-WebRequest?view=powershell-4.0

And returns an object out of which i only get the StatusCode property.

**Comment by rgauss** [comment]
Great work here.

I am a bit concerned that we have to remember to update the versions of Keycloak in both the Helm chart pulled in and `distribution/build.properties`, should we estimate the effort to develop checks for that either in the packaging script, bamboo build, or GitHub repo checks?

**Pull request merged**